README.md

```diff
@@ -1,62 +1,48 @@
-[![Conclave SDK](.github/assets/logo.png)][conclave_website]
+[![Conclave Core SDK](.github/assets/logo.png)][conclave_website]
 
-## Overview
-[Conclave SDK][docs] is an open source platform that makes working with SGX enclaves easy.
+The [Conclave Core SDK][docs] is an open source platform that lets you create SGX enclaves very easily. You can 
+write your enclave code in high-level languages such as Java, Kotlin and JavaScript.
```

Comment:
Please add the Oxford comma.
"write your enclave code in high-level languages such as Java, Kotlin, and JavaScript."